### ISSUE TITLE
Add a `plot()` method to the Queueing Network

### ISSUE THREAD

**Issue by drvinceknight** [geraintpalmer/ASQ#17]
This would tap in to the networkx code and produce a graphics object:

So something like:

```
>>> Q = asq.Simulation(params_dict)
>>> p = Q.plot()  # This would return a networkx/matplotlib plot
>>> p.show() 
>>> p.save() # etc...
```

Just a nice quick way to get a visualisation of the network? As it's already networkx underneath this would be easy enough to implement (I'd like to take this one). What do you think?

**Comment by geraintpalmer** [comment]
Sounds great! cool idea.

Would be nice to be able to denote number of servers, and if there's potential for blockages (ie if N1 -> N2, and N2 has limited capacity, then that arc has potential for blockages, etc.)

**Comment by drvinceknight** [comment]
Yeah I was thinking that. Shouldn't be a problem: just adding relevant
labels (node and edge) to the networkx class. I'll give it a go when I get
a chance.
